app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WelcomeController < ApplicationController
+
+  before_action :find_top_ten_movies, only: :index
+  before_action :find_top_ten_books, only: :index
+  before_action :find_top_ten_albums, only: :index
+  before_action :find_spotlight_work, only: :index
+
+  def index
+  end
+
+  private
+
+  def find_spotlight_work
+  @spotlight_work = Work.left_joins(:votes).group(:id).order('COUNT(votes.id) DESC').limit(1)
```

Comment:
Interesting... I think there could be a better way to find the work with the most votes. Also, please watch your indentation